Source/Diagnostics/ReducedDiags/ParticleHistogram2D.cpp

```diff
@@ -138,6 +138,7 @@ void ParticleHistogram2D::ComputeDiags (int step)
     Array<int,2> tlo{0,0}; // lower bounds
     Array<int,2> thi{m_bin_num_abs-1, m_bin_num_ord-1}; // inclusive upper bounds
     amrex::TableData<amrex::Real,2> d_data_2D(tlo, thi);
+    m_h_data_2D.clear();
     m_h_data_2D = amrex::TableData<amrex::Real,2> (tlo, thi, The_Pinned_Arena());
```

Comment:
```suggestion
    m_h_data_2D.resize(tlo, thi, The_Pinned_Arena());
```
This is more efficient.